core/templates/pages/exploration-editor-page/feedback-tab/services/thread-data-backend-api.service.ts

```diff
@@ -212,11 +222,12 @@ export class ThreadDataBackendApiService {
 
   async createNewThreadAsync(newSubject: string, newText: string):
     Promise<void | SuggestionAndFeedbackThreads> {
-    return this.http.post(this.getThreadListHandlerUrl(), {
-      state_name: null,
-      subject: newSubject,
-      text: newText
-    }).toPromise().then(async() => {
+    return this.http.post<void | SuggestionAndFeedbackThreads>(
```

Comment:
Why it can be `void`?

Reply:
Promise<void> will be returned when throwing an error